### PR TITLE
Add use case for settle

### LINKED ISSUE #219: Add in use case for settle in DG

### PR DESCRIPTION
**Pull request by Soumyadip-cmd**
Closes #219

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -401,6 +401,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. User requests to <ins>list students(UC02)</ins>.
+2. User enters command to settle outstanding fees for the specified student.
+3. System updates the total tuition fee paid and total tuition fee owed by the student.
+4. System displays success message.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The given index is invalid.
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md:417]: Can i check if using the word index is implementation specific? Furthermore, wouldn't the next extension catch this error as well?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -401,6 +401,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * Steps 2b1-2b2 are repeated until all details entered are correct.
 * Use cases resumes from step 3.
 
+**Use case: UC07 - Settle outstanding fees for student**
+
+**MSS**
+
+1. User requests to <ins>list students(UC02)</ins>.
```
Comment (Feng1231) [inline docs/DeveloperGuide.md:408]: Ordered list should be all `1.`

Comment (Feng1231) [inline docs/DeveloperGuide.md:408]: Ordered list should all be `1.`

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Refer to [here](https://se-education.org/guides/conventions/markdown.html)for ordered list format

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md:417]: I am not sure too. The other use cases also seem to use index. As for the next extension catching the error, I am unsure to whether lump all the errors together or to separate by types of errors

Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md]: I have removed the extension which uses index

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -401,23 +421,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-1. User enters command to find students.
-1. System displays list with students with matching details.
+1. User requests to <ins>list students(UC02)</ins>.
+1. User enters command to settle outstanding fees for the specified student. 
+1. System updates the total tuition fee paid and total tuition fee owed by the student.
+1. System displays success message.
 
    Use case ends.
 
 **Extensions**
```
Comment (TY1Fan) [inline docs/DeveloperGuide.md:435]: Could there be this possible extension where list returns nothing or student to settle payment is not in the list?

Comment (Soumyadip-cmd) [reply docs/DeveloperGuide.md:436]: Thank you for the suggestion. I have changed the usecase to use find and added in the extension where specified student is not found.

**Review by TY1Fan — APPROVED** [review]
LGTM

**Review by Feng1231 — APPROVED** [review]
LGTM

**Pull request merged**